india_stocks_api/brokers/angel.py

```
@@ -201,92 +442,516 @@ def get_quote(self, instrument: Equity | Future | Option | Index) -> dict:
+        except Exception as e:
+            return OrderExecutionResponse(
+                success=False,
+                broker="angel",
+                execution_status=ExecutionStatus.FAILED,
+                error=ErrorDetail(
+                    code="INTERNAL_ERROR",
+                    message=str(e),
+                ),
+            )
+
+    def modify_order(self, order_id: str, price: float = 0.0, trigger_price: float = 0.0, quantity: int = 0) -> OrderExecutionResponse:
```

Ruff (0.15.1)

[warning] Unused method argument: `trigger_price`

(ARG002)